notebooks/tutorials/hello-syft/01-hello-syft.ipynb

```diff
@@ -11,6 +11,45 @@
     "# Hello Syft"
```

Comment:
<p>I would create the dataset as one clal to make things shorter</p><p> </p><p> </p><pre class="ql-syntax" spellcheck="false">dataset = sy.Dataset(name="usa-mock-data",
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;asset_list=[sy.Asset(name="ages", data=pd.dataframe(...), mock=pd.daframe(...), mock_is_real=False)])
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/7775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>